rss2email/main.py

```diff
@@ -89,7 +89,7 @@ def run(*args, **kwargs):
         'email', nargs='?',
         help='target email for the new feed')
     add_parser.add_argument(
-        'from-email', nargs='?',
+        '--from-email', dest='from_email',
```

Comment:
`--` doesn’t look right

Reply:
It's the long form of this option. Without this, subparser consider it's a positional argument fetched into "from-email" member of args, which is inaccessible in Python.
I tried both 'from-email' and dest='from_email', but it complains that dest is supplied two times.
Now we keep the old behaviour with an optional argument which is, at end, not so bad.